src/main/PresentationExchange.ts

```diff
@@ -1,19 +1,15 @@
 import {
   EvaluationResults,
-  IPresentation,
   IPresentationDefinition,
-  IVerifiableCredential,
-  IVerifiablePresentation,
   KeyEncoding,
   PEX,
   PresentationSignCallBackParams,
   PresentationSignOptions,
-  ProofPurpose,
-  ProofType,
   SelectResults,
   Status,
 } from '@sphereon/pex';
 import { PresentationDefinitionV1, PresentationDefinitionV2, PresentationSubmission } from '@sphereon/pex-models';
+import { IPresentation, IProofPurpose, IProofType, IVerifiableCredential, IVerifiablePresentation } from '@sphereon/ssi-types';
```

Comment:
I'm not sure about this, we're adding a dependency to `SSI-SDK` here, but this project is also used by SSI-SDK. And also types can be something separate. maybe we can change the name of `pex-models` to `ssi-models` to include additional things.

Reply:
There is no dep on ssi-sdk itself. Ssi-types has no (transitive) deps. I quite deliberately made that choice to do it like that

Reply:
Also we are not going to use pex-models for that. As that is auto generated code

Reply:
Actually the whole point of having ssi-types and clearing ssi-core was to remove circular deps we had (core depending on siop, which depends on linked domain, which depends on core).

Both core and types now are clean in terms of deps. Meaning the circular deps are goneml. And I want ssi types to be in the mono repo of our sdk